modules/ROOT/pages/depl-examples/minimal-bare-metal.adoc

```diff
@@ -34,7 +37,17 @@ IMPORTANT: ownCloud highly recommends reading the xref:deployment/general/genera
 
 == Prerequisite
 
-See xref:prerequisites/prerequisites.adoc#filesystems-and-shared-storage[ocis prerequisites,window=_blank].
+See the xref:prerequisites/prerequisites.adoc#filesystems-and-shared-storage[Prerequisites,window=_blank] section for more details.
+
+== Used Settings
+
+The following settings were used in this guide. You can change this according your needs:
+
+* The Infinite Scale binary location: `{ocis_bin}` (OS default)
+* The Infinite Scale xref:deployment/general/general-info.adoc#configuration-directory[configuration directory]: `{ocis_env}`
+* The Infinite Scale xref:deployment/general/general-info.adoc#base-data-directory[base data directory]: `{ocis_data}`
+* The URL accessing Infinite Scale: `{ocis_url}`
+* The internal port accessing Infinite Scale: {ocis_port} (default)
```

Comment:
```suggestion
* The port for accessing Infinite Scale: {ocis_port} (default)
```

I don't think that the word "internal" is relevant here. The port number is used when accessing Infinite Scale from "outside" and from "inside".